aiohue/lights.py

```diff
@@ -61,6 +61,11 @@ def swversion(self):
         """Software version of the light."""
         return self.raw['swversion']
 
+    @property
+    def swupdateavailable(self):
```

Comment:
can we rename this to `swupdatestate`, to reflect the property names in the API?

Reply:
just renamed it